doc/classes/Node.xml

```diff
@@ -578,235 +608,234 @@
 		<member name="multiplayer" type="MultiplayerAPI" setter="" getter="get_multiplayer">
 			The [MultiplayerAPI] instance associated with this node. See [method SceneTree.get_multiplayer].
 		</member>
 		<member name="name" type="StringName" setter="set_name" getter="get_name">
-			The name of the node. This name is unique among the siblings (other child nodes from the same parent). When set to an existing name, the node will be automatically renamed.
-			[b]Note:[/b] Auto-generated names might include the [code]@[/code] character, which is reserved for unique names when using [method add_child]. When setting the name manually, any [code]@[/code] will be removed.
+			The name of the node. This name must be unique among the siblings (other child nodes from the same parent). When set to an existing sibling's name, the node is automatically renamed.
+			[b]Note:[/b] The following characters cannot be used: ([code].[/code] [code]:[/code] [code]@[/code] [code]/[/code] [code]"[/code] [code]%[/code]). In particular, the [code]@[/code] character is reserved for auto-generated names. See also [method String.validate_node_name].
 		</member>
 		<member name="owner" type="Node" setter="set_owner" getter="get_owner">
-			The node owner. A node can have any other node as owner (as long as it is a valid parent, grandparent, etc. ascending in the tree). When saving a node (using [PackedScene]), all the nodes it owns will be saved with it. This allows for the creation of complex [SceneTree]s, with instancing and subinstancing.
-			[b]Note:[/b] If you want a child to be persisted to a [PackedScene], you must set [member owner] in addition to calling [method add_child]. This is typically relevant for [url=$DOCS_URL/tutorials/plugins/running_code_in_the_editor.html]tool scripts[/url] and [url=$DOCS_URL/tutorials/plugins/editor/index.html]editor plugins[/url]. If [method add_child] is called without setting [member owner], the newly added [Node] will not be visible in the scene tree, though it will be visible in the 2D/3D view.
+			The owner of this node. When saving the owner node in a [PackedScene], all the nodes it owns are also saved with it. In the editor, nodes not owned by the scene root are usually not displayed in the Scene dock, and will [b]not[/b] be saved. To prevent this, remember to set the owner after calling [method add_child].
+			[b]Note:[/b] It's recommended to set the owner to an ancestor of this node. This is not required, and it can be useful when working with scene unique nodes (see [member unique_name_in_owner])
 		</member>
 		<member name="process_mode" type="int" setter="set_process_mode" getter="get_process_mode" enum="Node.ProcessMode" default="0">
-			Can be used to pause or unpause the node, or make the node paused based on the [SceneTree], or make it inherit the process mode from its parent (default).
+			The node's processing behavior. Can be changed to pause the [SceneTree]'s processing calls for this node. See [enum ProcessMode].
```

Comment:
> Can be changed to pause the SceneTree's processing calls for this node.

I think this is confusing.

1. Changing this property does not pause the calls. SceneTree skip processing calls to nodes that `can_process()`. In other words, it changes its own state, not the scene tree's state.
2. Input calls can also be paused, and the calls can be made not only from SceneTree, but also from Viewports.

Reply:
The concept of "processing" is very muddied across the virtual methods, description, SceneTree.xml, etc. etc. It may need a second look later but it does not hurt to make this less ambiguous.

Reply:
Take a look at the revision AND `can_process`'s description again.